.grit/patterns/no_unthrown_exceptions.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Exceptions should not be created without being thrown
+---
+
+# Exceptions should not be created without being thrown
+
+Creating a new Throwable without actually throwing it is useless and is probably due to a mistake.
```

Comment:
This is a bit dubious. At the least I think we'd want to also exclude cases where the throwable is assigned to a variable (which itself might be thrown).